examples/machine-controller.yaml

```diff
@@ -105,6 +105,7 @@ rules:
   - ""
   resources:
   - configmaps
+  - endpoints
```

Comment:
This is the wrong namespace, the `endpoints` are in the `default` namespace. This means we need another `clusterRole{,Binding}`